src/Pmi/Controller/WorkQueueController.php

```diff
@@ -356,6 +357,8 @@ public function exportAction(Application $app, Request $request)
             if ($hasFullDataAccess) {
                 $headers[] = 'Deactivation Status';
                 $headers[] = 'Deactivation Date';
+                $headers[] = 'gROR Consent Status';
```

Comment:
Should it have a lowercase `o` (e.g. `gRoR`) for consistency?

Reply:
There are some inconsistencies in the ticket itself. I'll check with bhinnata and finalize one. Thanks!